client/my-sites/plan-features/style.scss

```diff
@@ -911,8 +911,10 @@ body.is-section-signup #primary .segmented-control.is-customer-type-toggle {
 	position: relative;
 
 	.is-section-signup & {
-		width: 100vw;
-		margin-left: calc( 50% - 50vw );
+		--approximate-viewport-scrollbar-width: 20px;
+
+		width: calc( 100vw - var( --approximate-viewport-scrollbar-width ) );
+		margin-left: calc( 50% - 50vw + var( --approximate-viewport-scrollbar-width ) / 2 );
```

Comment:
See screenshot in PR description to understand what I'm trying to fix here.

Whereas the "real" width of scrollbars on macOS/Windows seems to be somewhere in the range 15-17px, I opted for a slightly larger number just to be safe (and to avoid having to calculate the scrollbar width dynamically).

Reply:
I know the intention is awesome, but IMHO we should not include other fixes (or trials of fixes) in the same Pull Request. My reasoning is as simple as:

What if we need to either Rollback this PR due to wrong logic in the plans, or wrong fix in the css?
Or what if we just want to rollback our adjustment, and the CSS fix seems to be consistent and solid?

Just my thoughts!

Reply:
I see your point, but my reason for including it was that it affects all variations of the Plans step, including the one we introduce here. So while this change isn't strictly necessary to achieve the primary goal with this PR, it still seemed directly related.

I'll update the testing instructions to clarify how it plays into the other changes.

We could also move it to a separate PR, I'm open to that. I guess if we think that it's a particularly sensitive change, it makes more sense to have a dedicated PR for it.

Reply:
This has been extracted into #67325